doc/source/docs/user_guide/aggregation_with_dissolve.rst

```diff
@@ -12,17 +12,21 @@ Aggregation with dissolve
 
 Spatial data are often more granular than we need. For example, we might have data on sub-national units, but we're actually interested in studying patterns at the level of countries.
 
-In a non-spatial setting, when all we need are summary statistics of the data, we aggregate our data using the ``groupby`` function. But for spatial data, we sometimes also need to aggregate geometric features. In the *geopandas* library, we can aggregate geometric features using the ``dissolve`` function.
+In a non-spatial setting, when all we need are summary statistics of the data, we aggregate our data using the :meth:`~pandas.DataFrame.groupby` function. But for spatial data, we sometimes also need to aggregate geometric features. In the *geopandas* library, we can aggregate geometric features using the :meth:`~geopandas.GeoDataFrame.dissolve` function.
 
-``dissolve`` can be thought of as doing three things: (a) it dissolves all the geometries within a given group together into a single geometric feature (using the ``unary_union`` method), and (b) it aggregates all the rows of data in a group using ``groupby.aggregate()``, and (c) it combines those two results.
+:meth:`~geopandas.GeoDataFrame.dissolve` can be thought of as doing three things:
 
-``dissolve`` Example
+(a) it dissolves all the geometries within a given group together into a single geometric feature (using the :attr:`~geopandas.GeoSeries.unary_union` method), and
+(b) it aggregates all the rows of data in a group using :ref:`groupby.aggregate <groupby.aggregate>`, and
+(c) it combines those two results.
+
+:meth:`~geopandas.GeoDataFrame.dissolve` Example
 ~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This line needs to have the same length as the text above.

Reply:
Sorry, I missed that. I fixed it just now in this [commit](https://github.com/geopandas/geopandas/pull/1931/commits/68a19888e6245533d04aef456d3344636addcc3f).